### PR TITLE
add centos 8

### PR DESCRIPTION
**Pull request by parkervcp**
This is a base install for centos 8.

Covers the Panel and Daemon.
Uses packages provided by redhat. (no extra repos other than docker.)

Many changes in CentOS 8 make this a much easier install.

### CODE REVIEW

File: community/installation-guides/daemon/centos8.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+In this guide we will install Pterodactyl's Daemon (wings) — including all of it's dependencies — and configure it to use a SSL connection.
+
+[[toc]]
+
+::: tip
+This guide is based off the [official installation documentation](/daemon/installing.md) but is tailored specifically for CentOS 8.
+:::
+
+## Install Requirements
+We will first begin by installing all of the Daemon's [required](/daemon/installing.md#dependencies) dependencies.
```
Comment (schrej) [inline community/installation-guides/daemon/centos8.md]: Maybe extend the link to `dependencies`.
```suggestion
We will first begin by installing all of the Daemon's [required dependencies](/daemon/installing.md#dependencies).
```

File: community/installation-guides/daemon/centos8.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+### General Requirements
+```bash
+dnf install -y tar unzip make gcc gcc-c++ python2
+
+alternatives --set python /usr/bin/python2
```
Comment (schrej) [inline community/installation-guides/daemon/centos8.md]: Why do we need python2? It's EOL soon, isn't it?

Comment (parkervcp) [reply community/installation-guides/daemon/centos8.md]: Needed to build mmagic

Comment (schrej) [reply community/installation-guides/daemon/centos8.md]: oh well

**Comment by schrej** [comment]
I haven't tested the guide, this is just regarding general from and (my limited) grammar.

**Pull request merged**